src/main/java/lotto/domain/Money.java

```diff
@@ -0,0 +1,41 @@
+package lotto.domain;
+
+import lotto.message.ExceptionMessage;
+
+public class Money {
+    private int amount;
+
+    public Money(int money) {
+        validateMoney(money);
+        this.amount = money;
+    }
+
+    private void validateMoney(int money) {
+        validateOverThanLottoPrice(money);
+        validateUnit(money);
+    }
+
+    private void validateOverThanLottoPrice(int money) {
```

Comment:
저는 validate라는 단어가 중복사용되는게 싫어서 validate 함수안에 예외처리 함수들에 대해서는 except라는 단어를 사용해서 메소드명을 짓는데 이에 대해서는 어떻게 생각하시나요! 사소하지만 잘못된 방식인지 의견을 들어보고싶습니다
예) 
```suggestion
    private void exceptOverThanLottoPrice(int money) 
```

Reply:
좋은 의견이네요! 어떤 방식이든 프로그램 내에서 통일만 해주면 정답은 없는 것 같습니다!
다만 보통 대부분 검증은 `validateXXX` 형식으로 많이들 지어서 다른 사람이 볼 때 검증 함수라는 것을 바로 알 수 있기 때문에 가독성에 조금이나마 도움을 주는 것 같아요!